docs/api/modules/Bundling-Modules.md

```diff
@@ -0,0 +1,186 @@
+
+- Applying a single custom webpack config for both your client and server builds using the `webpackConfigPath` option.
+- Applying different custom webpack configs for the client and server builds using `webpackClientConfigPath`, and `webpackServerConfigPath` options.
+
+Please see `one-app-bundlers`'s [`webpackConfigPath`, `webpackClientConfigPath`, & `webpackServerConfigPath`](https://github.com/americanexpress/one-app-cli/tree/master/packages/one-app-bundler#webpackconfigpath-webpackclientconfigpath--webpackserverconfigpath) for information.
+
+#### `performanceBudget`
+
+Used to set a custom [performance budget](https://webpack.js.org/configuration/performance/#performancemaxassetsize) for your client module build. The default value is 200e3.
+
+Please see `one-app-bundlers`'s [`performanceBudget`](https://github.com/americanexpress/one-app-cli/tree/master/packages/one-app-bundler#performancebudget) for information.
+
+#### `purgecss`
+
+This provides two options that can be passed to [purgecss](https://github.com/FullHuman/purgecss) to consider before stripping out CSS. The two options are:
+
+- `paths` - This is an array of additional paths that purgecss should strip CSS from.
+
+- `disabled` - This used to disable purgecss from stripping out CSS. Disabling this would have a negative impact on performance.
+
+Please see `one-app-bundlers`'s [`purgecss`](https://github.com/americanexpress/one-app-cli/tree/master/packages/one-app-bundler#purgecss-options) for information.
+
+### CLI Commands
+
+`one-app` modules consists of [NPM Script Commands](https://docs.npmjs.com/misc/scripts) that are used to bundle and test the module. These scripts are provided from generating a module with [generator-one-app-module](https://github.com/americanexpress/one-app-cli/tree/master/packages/generator-one-app-module) and can be expanded on depending on the functionality that is needed.
+
+```sh
+npm run <one-app-module-npm-command> 
+# e.g. npm run build
+```
+
+We describe each command and a description of its usage below. 
+
+**Contents**
+
+* Build Commands
+  * [`prepare`](#build-commands)
+  * [`prebuild`](#build-commands)
+  * [`build`](#build-commands)
+  * [`watch:build`](#build-commands)
+* Clean Commands
+  * [`clean`](#clean-commands)
+* Lint Commands
+  * [`lint`](#lint-commands)
+  
+## Build Commands
+
+| Command       | Description                                                                                    |
+|---------------|------------------------------------------------------------------------------------------------|
+| `prepare` | Runs `build` before the the module is packed and published. |
+| `prebuild` | Cleans the build directory before a build is done. |
+| `build` | Builds the `one-app` modules.|
+| `watch:build` | Builds the `one-app` modules and watches source files with [`nodemon`](https://nodemon.io/).When source changes, the `one-app` module  runs `npm run build`. *See [`build` Usage](#build-usage).* |
```

Comment:
Missing a space between `.` and `When`

Reply:
Also an extra space between `module` and `runs`

Reply:
thanks for catching this updated c5b191e